packages/cli/src/export/__tests__/exportJsonAssetsToCsv.test.ts

```diff
@@ -203,7 +203,7 @@ test('should log error when export fails', async () => {
       storageDirectory: TEST_DIRECTORY,
       apiKey: TEST_API_KEY,
     }),
-  ).rejects.toThrow(/Unexpected token/g);
+  ).rejects.toThrow();
```

Comment:
JSON parsing error message changes in later verison.

Reply:
I'm interested to see if it actually improved!